src/app/systelab-components/datepicker/datepicker-time.component.ts

```diff
@@ -7,19 +7,19 @@ import { I18nService } from 'systelab-translate/lib/i18n.service';
 	selector: 'systelab-date-time',
 	templateUrl: 'datepicker-time.component.html'
 })
-export class DatepickerTime extends Datepicker {
+export class DatepickerTimeComponent extends Datepicker {
 
 	public touchSpinHourValues: TouchSpinValues;
-	public touchSpinMinuteValues: TouchSpinValues;
+	public touchSpinMinutesValues: TouchSpinValues;
 
-	protected _currentHours: string;
-	protected _currentMinutes: string;
+	@Output() public currentHoursChange = new EventEmitter<number>();
```

Comment:
Maybe we can remove these two outputs, we can review if someone is using them, or mark them as deprecated

Reply:
Done 👍